### PR TITLE
Add all 'add to homescreen' assets to the project

### LINKED ISSUE #3588: Add all 'add to homescreen' assets to the project
**Is your feature request related to a problem? Please describe.**
All the `add to homescreen` assets for all services need to be added to the project.

**Describe the solution you'd like**
Add the assets for the different services at once, in one PR. Also confirm links to these assets are correct in the manifest.json and that the link to the manifest is correct - not in the articles directory. 

**Describe alternatives you've considered**
N/A.

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
The assets can be found here -- https://www.dropbox.com/sh/4eljarf21j4p2hl/AADOfRYB4byBdzB4PpoMvSXza?dl=0

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #3588 

**Overall change:** 
Added assets for all of the services to the project.

**Code changes:**

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by AlistairGempf — APPROVED** [review]
Looks good to me

**Review by EinsteinNjoroge — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

File: src/app/lib/config/services/afaanoromoo.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ const service = {
     noBylinesPolicy: 'https://www.bbc.com/news/help-41670342#authorexpertise',
     publishingPrinciples: 'https://www.bbc.com/news/help-41670342',
     script: latin,
-    manifestPath: '/manifest.json',
+    manifestPath: '/afaanoromoo/manifest.json',
```
Comment (dr3) [inline src/app/lib/config/services/afaanoromoo.js]: These all need to be `'/manifest.json'` not `'/afaanoromoo/manifest.json'` 🙈

**Review by dr3 — CHANGES_REQUESTED** [review]

File: dataValidator/helpers/dataLoader/asyncValidateDir.test.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ describe('asyncValidateDir helper', () => {
 
   it('should call readScenario for every file in the /data directory', async () => {
     const readScenarioSpy = jest.spyOn(readScenario, 'readScenario');
-    await expectMethodToBeCalledTimes(291, readScenarioSpy);
+    await expectMethodToBeCalledTimes(293, readScenarioSpy);
```
Comment (sareh) [inline dataValidator/helpers/dataLoader/asyncValidateDir.test.js]: Tests are currently failing on this value.
```suggestion
    await expectMethodToBeCalledTimes(291, readScenarioSpy);
```

**Review by dr3 — APPROVED** [review]
LGTM :)

**Comment by staylos92** [comment]
LGTM 👍 

Would need to check this again when the assets are live to ensure the icons are all being picked up correctly.

**Pull request merged**